plugins/advisors/osv/src/main/kotlin/Osv.kt

```diff
@@ -143,16 +142,7 @@ private fun createRequest(pkg: Package): VulnerabilitiesForPackageRequest {
     // TODO: Support querying vulnerabilities by Git commit hash as described at https://osv.dev/docs/#section/OSV-API.
     //       That would allow to generally support e.g. C / C++ projects that do not use a dedicated package manager
     //       like Conan.
-
-    // Work-around for missing purl converters, see https://github.com/google/osv.dev/issues/2402.
-    return if (pkg.id.type == "Swift") {
-        VulnerabilitiesForPackageRequest(
-            pkg = org.ossreviewtoolkit.clients.osv.Package(name = pkg.id.name, ecosystem = Ecosystem.SWIFT_URL),
-            version = pkg.id.version
-        )
-    } else {
-        VulnerabilitiesForPackageRequest(pkg = org.ossreviewtoolkit.clients.osv.Package(purl = pkg.purl))
-    }
+    return VulnerabilitiesForPackageRequest(pkg = org.ossreviewtoolkit.clients.osv.Package(purl = pkg.purl))
```

Comment:
Should this function be turned into an expression while at it ?

Reply:
Actually, let me just inline it instead.